core/src/main/java/com/linecorp/armeria/server/http/dynamic/DynamicHttpFunctionEntry.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.http.dynamic;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.EnumSet;
+import java.util.Map;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.Sets;
+
+import com.linecorp.armeria.common.http.HttpMethod;
+
+/**
+ * {@link HttpMethod}, {@link DynamicPath} and their corresponding {@link DynamicHttpFunction} instance.
+ */
+final class DynamicHttpFunctionEntry {
+
+    /**
+     * Set of HTTP Method, e.g., GET, POST, PUT, ...
+     */
+    private final EnumSet<HttpMethod> methods;
```

Comment:
No, I meant 'please change the type of `methods` from `EnumSet<HttpMethod>` to `Set<HttpMethod>`. I don't see a reason to declare it as an `EnumSet`.